torch/fx/experimental/Partitioner.py

```diff
@@ -81,16 +93,18 @@ class Partitioner:
     along with a new graph module with partitions as submodule nodes.
     """
     def __init__(self) -> None:
-        self.partitions: Set[Partition] = set()
+        self.partitions: List[Partition] = []
         self.devices: List[Device] = []
-        self.node_to_partitions: Dict[Node, List[int]] = {}
-        self.partition_to_used_mem_bytes: Dict[Partition, int] = {}
+        self.node_to_partitions: Dict[Node, int] = {}
+        self.embedding_partition_indexes: Set[int] = set()
+        self.num_of_logical_devices: int = 0
 
     def partition_graph(
         self,
         fx_module: GraphModule,
         torch_module: torch.nn.Module,
-        devices: List[Device]
+        devices: List[Device],
+        is_sparse_nn: bool = False
```

Comment:
Can you wrap this in a class? With future schemes coming this won't scale adding a new parameter each time. 
Like we have compilationContext in glow, Could you add a PartitionerConfig? This could be passed in at init()